PowerEditor/src/ScintillaComponent/Buffer.cpp

```diff
@@ -868,15 +869,14 @@ bool FileManager::backupCurrentBuffer()
 				::SetFileAttributes(fullpath, dwFileAttribs);
 			}
 
-			FILE *fp = UnicodeConvertor.fopen(fullpath, TEXT("wbc"));
-			if (fp)
+			if (UnicodeConvertor.fopen(fullpath))
 			{
 				int lengthDoc = _pNotepadPlus->_pEditView->getCurrentDocLen();
 				char* buf = (char*)_pNotepadPlus->_pEditView->execute(SCI_GETCHARACTERPOINTER);	//to get characters directly from Scintilla buffer
-				size_t items_written = 0;
+				long items_written = 0;

```

Comment:
…as then it seems this switching from `size_t` to `long` and later casts won't be necessary.